src/ontology/catalog-v001.xml

```diff
@@ -1,201 +1,11 @@
-        <uri name="http://purl.obolibrary.org/obo/uberon/imports/fbdv_import.owl" uri="imports/fbdv_import.owl"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/imports/fbdv_import.obo" uri="imports/fbdv_import.obo"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/imports/ma_import.owl" uri="imports/ma_import.owl"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/imports/ma_import.obo" uri="imports/ma_import.obo"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/imports/poro_import.owl" uri="imports/poro_import.owl"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/imports/poro_import.obo" uri="imports/poro_import.obo"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/imports/wbbt_import.owl" uri="imports/wbbt_import.owl"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/imports/wbbt_import.obo" uri="imports/wbbt_import.obo"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/imports/wbls_import.owl" uri="imports/wbls_import.owl"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/imports/wbls_import.obo" uri="imports/wbls_import.obo"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/imports/xao_import.owl" uri="imports/xao_import.owl"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/imports/xao_import.obo" uri="imports/xao_import.obo"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/imports/zfa_import.owl" uri="imports/zfa_import.owl"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/imports/zfa_import.obo" uri="imports/zfa_import.obo"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/mod/uberon-combined-adult-mammal.owl" uri="bridge/uberon-combined-adult-mammal.owl"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/mod/uberon-combined-amniote.owl" uri="bridge/uberon-combined-amniote.owl"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/mod/uberon-combined-embryonic-mammal.owl" uri="bridge/uberon-combined-embryonic-mammal.owl"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/mod/uberon-combined-eukaryote.owl" uri="bridge/uberon-combined-eukaryote.owl"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/mod/uberon-combined-mammal.owl" uri="bridge/uberon-combined-mammal.owl"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/mod/uberon-combined-metazoa.owl" uri="bridge/uberon-combined-metazoa.owl"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/mod/uberon-combined-opisthokont.owl" uri="bridge/uberon-combined-opisthokont.owl"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/mod/uberon-combined-teleost.owl" uri="bridge/uberon-combined-teleost.owl"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/mod/uberon-combined-tetrapod.owl" uri="bridge/uberon-combined-tetrapod.owl"/>
-        <uri name="http://purl.obolibrary.org/obo/uberon/mod/uberon-combined-vertebrate.owl" uri="bridge/uberon-combined-vertebrate.owl"/>
```

Comment:
Are you sure the above are not needed for composite metazoan? What is the reason to delete them all?